configure.ac

```diff
@@ -1092,14 +1092,14 @@ AS_CASE(["$arch,$system"],
       [clang-*],
         [default_as='clang -arch x86_64 -Wno-trigraphs -c'
         default_aspp='clang -arch x86_64 -Wno-trigraphs -c'],
-      [default_as="${toolpref}as -arch x86_64"
+      [default_as="${toolpref}gcc -arch x86_64 -c"
       default_aspp="${toolpref}gcc -arch x86_64 -c"])],
   [amd64,solaris],
     [AS_CASE([$ocaml_cv_cc_vendor],
       [sunc-*],
         [AS_IF([test x"$SYSTEM_AS" = "x"],
           [AC_MSG_ERROR([GNU as assembler is required.])],
-          [default_as="${toolpref}as --64"
+          [default_as="${toolpref}gcc -m64 -c"
```

Comment:
In this case we tested for `as` above - don't we need to test for `gcc` as well? cf. difference between `default_as` and `default_aspp` here?